bin/extract-ORFs.py

```diff
@@ -64,19 +86,59 @@ def translations(read):
         def translations(read):
             return read.translations()
 
+    if kozakInfo or kozakOnly:
+        if aa:
+            # Should the code break in this case?
+            print('Did you pass NA sequences? Kozak consensuses can only'
```

Comment:
If `aa` is true, then yes they must have specified on the command line that they have AA data. So I'd print an error and also exit. Don't ask if they passed AA (NA?), just tell them the code can't be used on AA seqs.

Reply:
done